### PR TITLE
Ensure inventory supports multi-hypervisor

### PR DESCRIPTION
**Pull request by cjeanner**
With #952, we add the capability to get multiple hypervisor.

We therefore need to ensure the inventory is properly generated to allow
accesses from the controller-0.

Note that specific documentation about this feature will come shortly,
in a dedicated patch, with proper example for the hypervisor setup.

As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running

### CODE REVIEW

**Review by rebtoor — APPROVED** [review]

**Review by rebtoor — APPROVED** [review]

File: ci_framework/roles/reproducer/tasks/configure_controller.yml [hunk 1]
```diff
@@ -88,6 +87,49 @@
...
+        content: "{{ _ssh_key }}"
+        mode: "0600"
+        owner: "zuul"
+        group: "zuul"
+      loop: "{{ hostvars.keys() }}"
+      loop_control:
+        label: "{{ host }}"
+        loop_var: "host"
+
+    - name: Inject ProxyHost configuration
```
Comment (pablintino) [inline ci_framework/roles/reproducer/tasks/configure_controller.yml:111]: Is this name/task code enough to explain why this task is needed? I know how the PR works cause you explain me the idea beforehand but I hardly doubt that I'm gonna be able to understand why is this needed and why is this mentioning ProxyHost without doing anything with/to it in the SSH config file.

Comment (cjeanner) [reply ci_framework/roles/reproducer/tasks/configure_controller.yml:111]: right - might be worth a comment or two. Task name on its own is correct - but getting some context in comments may be good.

**Comment by cjeanner** [comment]
recheck

**Review by raukadah — COMMENTED** [review]
/approve

**Review by raukadah — APPROVED** [review]
